.gitlab-ci.yml

```diff
@@ -371,6 +372,7 @@ Publish CP podspecs (legacy):
     - !reference [.release-pipeline-40m-delayed-job, rules]
```

Comment:
Same question here with the 40 min delay

Reply:
Same as above